sdk/data/aztables/internal/zz_service_client.go

```diff
@@ -0,0 +1,230 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) AutoRest Code Generator. DO NOT EDIT.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+package internal
+
+import (
+	"context"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+// ServiceClient contains the methods for the Service group.
+// Don't use this type directly, use a constructor function instead.
+type ServiceClient struct {
+	internal *azcore.Client
+	endpoint string
+	version  Enum0
```

Comment:
Can we get a better name on Enum0

Reply:
I can do that in a separate PR so that the cleanup doesn't get lost in the noise of regeneration.

Reply:
It is internal only though, so having a better name isn't a huge deal IMO.